examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,64 @@
+"""
+=====================================
+Plotting points on a Map with WCSAxes
+=====================================
+
+This example demonstrates the plotting of a point, a line and an arch in pixel coordinates,
+world coordinates, and SkyCoords respectively when plotting a map with WCSAxes.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.utils import GreatArc
+
+###############################################################################
+# We will start by creating a map using an AIA 171 image.
+# Now we will plot a line on the map by using coordinates in arcseconds.
+# The array below `xx` and `yy` are the x and y coordinates that define a
+# line from the Sun center (at 0, 0) to the point (500, 500) in arcsecs.
+# Plotting with WCSAxes expects pixel coordinates, but we can plot
+# world coordinates (i.e. arcsec) by using the ``transform`` keyword.
```

Comment:
```suggestion
# When plotting a map a WCSAxes is created. 
# For plotting with WCSAxes, pixel coordinates are expected as a default, however, we can plot world coordinates (i.e. arcsec) by using the ``transform`` keyword.
# Its important to note that when transforming between world and pixel coordinates
# the world coordinates need to be in degrees rather than arcsecs.
```